backend/src/main/java/fk/prof/backend/model/association/impl/ZookeeperBasedBackendAssociationStore.java

```diff
@@ -1,11 +1,12 @@
 package fk.prof.backend.model.association.impl;
 
-import fk.prof.backend.exception.AssociationException;
+import fk.prof.backend.exception.BackendAssociationException;
```

Comment:
Agreed. can we hold on package movement for a bit. There are couple of branches waiting to be merged on top of this once this goes to master (considering we are racing to release, gaurav and me have forked separate branches on top of this one to stay unblocked) and merging will become a pain if we do lot of movement.